meteor/client/ui/Settings/components/triggeredActions/actionEditors/actionSelector/ActionSelector.tsx

```diff
@@ -162,6 +167,8 @@ function actionToLabel(t: TFunction, action: SomeAction['action']): string {
 			return t('Go to On Air line')
 		case ClientActions.showEntireCurrentSegment:
 			return t('Show entire On Air Segment')
+		case ClientActions.miniShelfQueueAdLib:
+			return t('MiniShelf queue AdLib')
```

Comment:
I think we had a disussion about _queue_ vs _cue_ and _cue_ won. I actually wonder if this should be renamed "select", since _cue_ suggests a playout operation being performed, while this seems to be an entirely client-side operation?

Reply:
It is consistent with `isQueueable`, `innerStartOrQueueAdLibPiece` and so on. It is not just a client-side operation. When triggered, it executes the START_ADLIB userAction and may result in queueing a Part.

Reply:
We're pretty inconsistent atm with queue vs cue anyway, so we'll leave this for now.

Reply:
If this executes `START_ADLIB` then this needs to be documented somewhere. I wouldn't expect a "queue" action to "start" something.